wrapper_client/src/main.cpp

```diff
@@ -213,61 +213,16 @@ SDL_AppResult SDL_AppInit(void** appstate, int argc, char* argv[]) {
 
     Logger::getInstance()->log("Logger started successfully!");
 
-    PlayerInput* input = new SDLInput(getGamepad());
-
-    // Initialize AudioManager
-    AudioManager* audio = new SDL3AudioManager(); // Pass basePathSOS to constructor
+    PlayerInput* input = new SDLInput(getGamepad());    // Initialize AudioManager
+    AudioManager& audio = SDL3AudioManager::Instance();
+    // Set the global AudioManager instance to point to our SDL3AudioManager implementation
+    AudioManager::SetInstance(&audio);
     
-
-    if(audio->initialize(basePathSOS.string())){
+    if(audio.initialize(basePathSOS.string())){
```

Comment:
If you do something like

```suggestion
auto basePathAudio = basePathSOS / "SOS/assets";
 if(audio.initialize(basePathAudio.string())){
```


You don't need to specify SOS/assets afterwards